doc/user/content/release-notes.md

```diff
@@ -19,6 +19,69 @@ the PR description instead. They will be migrated here during the release
 process by the release notes team.
 {{< /comment >}}
 
+{{% version-header v0.22.0 %}}
+
+- **Breaking change.** Standardize handling of the following [nonpure functions](/sql/functions/#pure-and-nonpure-functions) {{% gh 10445 %}}:
+
+  - `current_database`
+  - `current_timestamp`
+  - `current_role`
+  - `current_schema`
+  - `current_schemas`
+  - `current_user`
+  - `mz_cluster_id`
+  - `mz_logical_timestamp`
+  - `mz_uptime`
+  - `mz_version`
+  - `session_user`
+  - `pg_backend_pid`
+  - `pg_postmaster_start_time`
+  - `version`
+
+  Materialize now allows use of nonpure functions in views, but will refuse to
+  create an index that directly or indirectly depends on a nonpure function.
+  The one exception is [`mz_logical_timestamp`], which can be used in limited
+  contexts in a materialized view as a [temporal filter](/guides/temporal-filters).
+
+  Previously `current_timestamp`, `mz_logical_timestamp`, and `mz_uptime` were
+  incorrectly disallowed in unmaterialized views, while the remaining nonpure
+  functions were incorrectly allowed in materialized views.
+
+- **Breaking change.** Store days separately in [`interval`].
+Unlike the previous version, Hours are never converted to days. This
+means that: an Interval of 24 hours will not be equal to an Interval
+of 1 day, you cannot subtract hours from days, and when ordering
+Intervals `d days > h hours` for all `d,h` {{% gh 10708 %}}.
+
+- **Breaking change.** Print all negative [`interval`]
+units as plural (e.g., `-1 days` will be printed instead of `-1 day`). This
+matches the behavior of PostgreSQL.
+
+- **Breaking change.** Round microsecond field of
+[`interval`] to 6 places before applying the
+given precision. For example `INTERVAL '1.2345649' SECOND(5)` will be
+rounded to `00:00:01.23457`, not `00:00:01.23456`. This matches the
+behavior of PostgreSQL.
+
+- Add the following aliases to Interval parsing: `yr`, `yrs`, `hr`, `hrs`, `min`, `mins`, `sec`, `secs`.
```

Comment:
```suggestion
- Add several new time units to [`interval`](/sql/types/interval) parsing: `yr`, `yrs`, `hr`, `hrs`, `min`, `mins`, `sec`, and `secs`.
```

Naturally, we forgot to update the docs, too: https://preview.materialize.com/10991/sql/types/interval/. The time units are listed in the time-unit.svg diagram.

Reply:
On my list.